assets/src/components/cd/clusters/ClusterUpgradePR.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react'
+import { Button } from '@pluralsh/design-system'
+
+import { useCreatePullRequestMutation } from '../../../generated/graphql'
+
+export function ClusterUpgradePR({ prs, setError }) {
+  const pr = prs[0]
+  const [mutation, { loading, error }] = useCreatePullRequestMutation({
+    variables: {
+      id: pr.id,
+      branch: 'mjg/upgrade',
```

Comment:
Is this supposed to be hardcoded?

Reply:
It was for a demo a while ago, we can get rid of it in another pr if we want